doc/usage/vscode-extensions.md

```diff
@@ -0,0 +1,37 @@
+# Add VSCode extensions in HuronOS
+So, you want to add a vscode extension to the huronOS, follow this guide and you'll succeed
+
+## 1) Getting the extension folder
+For this you have two ways
+
+1) From the app
+    - Download the extension from the vscode app
+    - Locate the extension folder in the following path
+        ```
+        # Vscode
+        ~/.vscode/extensions
+        # Vscodium
+        ~/./vscode-oss/extensions
+        ```
+
+2) From the extensions marketplace
+    - Download the .vsix of the extension
+    - Unpackage the extension
+    - Rename the "extension" folder to the name of the extension
+
+## Create the module
+Now with the folder located and assumming
+- you're on an HuronOS instance
+- or you have access to the dir2hsm script
+- and the extension is located in the current path
+
+execute the following commands:
+
+
+```
+# Replace "my_extension" with the name you want to identify the extension
+$ mkdir -p my_extension.hsm/home/contestant/.vscode-oss/extensions/
+$ mv -r my_extension my_extension.hsm/home/contestant/.vscode-oss/extensions/
```

Comment:
Can you check this [workaround](https://github.com/microsoft/vscode/issues/56614#issuecomment-754839933) for making the extensions user-global instead of using the user-specific config directory? 
It would be preffered to use the less possible the contestant directory as that could require a `chown` step as seen before with the JetBrains packages.